_communication/Admission to Temasek Primary.md

```diff
@@ -5,6 +5,6 @@ description: ""
 ---
 Application for 2024 waitlist is now open and will close on 15 November 2023. 
 
-If you would like to apply to be on our waitlist for Primary 2 to 6 for Academic Year 2024, please fill in the waitlist form below and submit it together with the supporting documents to Temasek_ps@moe.edu.sg by 15 November 2023.
+If you would like for your child to be on our waitlist for Primary 2 to 6 for Academic Year 2024, please fill in the waitlist form below and submit it together with the supporting documents to Temasek_ps@moe.edu.sg by 15 November 2023.
```

Comment:
If you would like for your child to be on our waitlist for Primary 2 to 6 for Academic Year 2024, please fill in the waitlist form below and submit it together with the supporting documents to Temasek_ps@moe.edu.sg by 20 November 2023.